receiver/oracledbreceiver/scraper.go

```diff
@@ -269,10 +269,17 @@ func (s *scraper) scrape(ctx context.Context) (pmetric.Metrics, error) {
 			now := pcommon.NewTimestampFromTime(time.Now())
 			for _, row := range rows {
 				tablespaceName := row["TABLESPACE_NAME"]
-				err := s.metricsBuilder.RecordOracledbTablespaceSizeLimitDataPoint(now, row["VALUE"], tablespaceName)
-				if err != nil {
-					scrapeErrors = append(scrapeErrors, err)
+				var val int64
+				if row["VALUE"] == "" {
```

Comment:
Would it be worth making "VALUE" a variable instead of just plain string throughout this method? It's used a few times at least so it might be nice.

Reply:
Indeed. Fixing.